docs/monitoring/metrics.md

```diff
@@ -1341,11 +1341,16 @@ Metrics related to data exchange between task executors using netty network comm
       <td>Gauge</td>
     </tr>
     <tr>
-      <th rowspan="1">Task</th>
+      <th rowspan="2">Task</th>
       <td>checkpointAlignmentTime</td>
       <td>The time in nanoseconds that the last barrier alignment took to complete, or how long the current alignment has taken so far (in nanoseconds).</td>
       <td>Gauge</td>
     </tr>
+    <tr>
+      <td>checkpointStartDelay</td>
+      <td>The time in nanoseconds that elapsed between the creation of the last checkpoint and the time when the checkpointing process has started by this Task. This delay shows how long it takes for a first checkpoint barrier to reach the task. Back-pressure will increase this value.</td>
```

Comment:
This delay shows how long it takes for **the** first checkpoint barrier to reach the task.
A high value indicates back-pressure. If only a specific task has a long start delay, the most likely reason is data skew.

Reply:
Just to make sure that this was not just an oversight:
I also suggested to add "A high value indicates back-pressure. If only a specific task has a long start delay, the most likely reason is data skew." instead of "Back-pressure will increase this value."

Reply:
Ops, sorry I've missed that. Updating.